server/node/routes.js

```diff
@@ -53,10 +53,22 @@ router.post('/payment_intents', async (req, res, next) => {
   const amount = await calculatePaymentAmount(items);
 
   try {
+    //build initial payment methods which should exclude currency specific ones
+    let initPaymentMethods = config.paymentMethods.filter(function(value, index, arr){
+      let includePaymentMethod = true;
+      switch(value)
+      {
+        case 'au_becs_debit':
+          includePaymentMethod = false;
+          break;
+      }
+      return includePaymentMethod;
+    });
```

Comment:
It might be more idiomatic to do something like this:

```javascript
const initPaymentMethods = config.paymentMethods.filter((value, index, arr) => {
  return ['au_becs_debit'].includes(value);
});
```

Reply:
Great suggestion! Much cleaner!